app/components/input-identity/component.js

```diff
@@ -81,13 +81,16 @@ export default Ember.Component.extend({
   }.property('allIdentities.@each.{logicalType,externalIdType}','allowTeams'),
 
   dropdownLabel: function() {
+    let out = '';
+    let intl = this.get('intl');
     if ( this.get('access.provider') === 'githubconfig' )
     {
-      return "Your Teams and Organizations";
+      out = intl.findTranslationByKey('inputIdentity.dropdownLabel.teams');
```

Comment:
Pls use `intl.t('translation.key')` anywhere you have `findTranslationByKey` (or `intl.tHtml('translation.key')` if HTML is needed).  There's no *actual* difference in this case but `findTranslationByKey` just returns the raw translation string, it won't interpolate variables or do pluralization, etc.